lineman/app/models/discussion_model.coffee

```diff
@@ -118,3 +100,19 @@ angular.module('loomioApp').factory 'DiscussionModel', (BaseModel) ->
     maxLoadedSequenceId: ->
       item = _.max @events(), (event) -> event.sequenceId or 0
       item.sequenceId
+
+    changeVolume: (volume) ->
```

Comment:
The permissions for update discussion are really different to those for set volume. For instance I can't update all the discussions I can read. I reckon we'll need a special endpoint for setting volume.